packages/theme/getStore.ts

```diff
@@ -0,0 +1,58 @@
+import axios from 'axios';
+import urlJoin from 'url-join';
+
+interface StoreConfig {
+  backendUrl: string
+}
+
+/* eslint-disable camelcase */
+export interface StoreAttr {
```

Comment:
Normally, when deserializng, we're transforming the API response. Consider merging IStore and StoreAttr to one interface. Pass the api response directly to the deserializer function (deserializeStore).

Reply:
Makes sense :)